src/components/shared/basicGrid/basicCell.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'm not a fan of the name `basic`. Seems like this is generalizing too early. I'd say just go with PropertyGrid or something similar and put in back with the deviceDetails grid folder structure.

Reply:
I don't like the name either.  Since I'm planning to use it in the other flyouts, it doesn't feel too early.  But if you don't want it general now, I'll move it back under deviceDetails for now.